src/pyqasm/maps/__init__.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2024 qBraid
+#
+# This file is part of pyqasm
+#
+# Pyqasm is free software released under the GNU General Public License v3
+# or later. You can redistribute and/or modify it under the terms of the GPL v3.
+# See the LICENSE file in the project root or <https://www.gnu.org/licenses/gpl-3.0.html>.
+#
+# THERE IS NO WARRANTY for pyqasm, as per Section 15 of the GPL v3.
+
+"""
+Module mapping supported QASM gates/operations to lower level gate operations.
+
+"""
+
+from typing import Callable, Union
+
+from openqasm3.ast import (
+    ClassicalDeclaration,
+    QuantumBarrier,
+    QuantumGate,
+    QuantumGateDefinition,
+    QuantumMeasurementStatement,
+    QuantumReset,
+    QubitDeclaration,
+    SubroutineDefinition,
+)
+
+# Define the type for the operator functions
+OperatorFunction = Union[

```

Comment:
Why do we need `OperatorFunction` in this file? Better to move to `expressions.py` as only that file is using it

Reply:
Done